lib/utils/fs.go

```diff
@@ -280,6 +331,9 @@ func overwriteFile(filePath string) (err error) {
 	}
 
 	_, err = io.CopyN(f, rand.Reader, size)
+
+	f.Sync() // sync to ensure commit to hardware
```

Comment:
Explicitly ignore error.

```suggestion
	_ = f.Sync() // sync to ensure commit to hardware
```